### DISCUSSION TITLE
Hiring.

### DISCUSSION THREAD

**Discussion by exploits** [wraith-labs/wraith#4] [General]
Hey, I'm interested in hiring a socket developer for my own project, do you have XMPP/Jabber?

**Comment by 0x1a8510f2** [comment]
I do not, unfortunately. If you wish to get in touch privately, I'm reachable on Matrix as [@0x1a8510f2:0x1a8510f2.space](https://matrix.to/#/@0x1a8510f2:0x1a8510f2.space) or over email at admin@0x1a8510f2.space.
